### PR TITLE
Clean up: ORCID

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions
- (re)moved schema from miscellaneous to new folder called digitalIdentifier for better grouping and decluttering of the miscellaneous folder

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

File: schemas/digitalIdentifier/ORCID.schema.tpl.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "identifier": {
       "type": "string",
       "pattern": "^https:\/\/orcid.org\/[0-9]{4}-[0-9]{4}-[0-9]{4}-([0-9]{3}[A-Z]|[0-9]{4})$",
-      "_instruction": "Enter the resolvable identifier (IRI) of the Open Researcher and Contributor ID, Inc."
+      "_instruction": "Enter the identifier for researchers provided by the Open Researcher and Contributor ID, Inc. (ORCID) as an internationalized resource identifier (IRI) following the defined pattern (i.e., 'https://orcid.org/' + ORCID)."
```
Comment (lzehl) [inline schemas/digitalIdentifier/ORCID.schema.tpl.json]: ... by Open Researcher and Contributor ID, Inc. (ORCID, Inc.)

Comment (lzehl) [reply schemas/digitalIdentifier/ORCID.schema.tpl.json]: @apdavison please correct if this suggestion is incorrect

**Review by lzehl — CHANGES_REQUESTED** [review]

**Review by lzehl — APPROVED** [review]
changed the instructions slightly according to my previous comment

**Comment by lzehl** [comment]
@Peyman-N or @aeidi89 or @apdavison feel free to merge is you also approve now

**Review by aeidi89 — APPROVED** [review]
Looks good.

**Review by Peyman-N — APPROVED** [review]

**Pull request merged**